themes/theme-gmd/locale/cs-CZ.json

```diff
@@ -0,0 +1,326 @@
+{
+  "common": {
```

Comment:
changes from the `@shopgate/user-privacy` extension also needs to be added to this repo.

https://github.com/shopgate/ext-user-privacy/pull/13

Reply:
added